cafebabel/articles/tags/views.py

```diff
@@ -46,7 +47,13 @@ def edit(slug):
     tag = Tag.objects.get_or_404(slug=slug, language=current_language())
 
     if request.method == 'POST':
-        tag.save_from_request(request)
+        try:
+            tag.save_from_request(request)
+        except ValidationError as e:
+            # TODO: see https://github.com/cafebabel/cafebabel.com/issues/187
+            flash('There was an error in your tag submission:')
+            flash(str(e))
```

Comment:
Do we actually want 2 splitted error messages here?
Also it should be [flagged with the _error_ category](http://flask.pocoo.org/docs/0.12/patterns/flashing/#flashing-with-categories)

Reply:
Nice! I'll flag and merge (note that it has to be handled on the front side)

Reply:
We [use flask-security messages template](https://github.com/cafebabel/cafebabel.com/blob/master/cafebabel/templates/base.html#L70) that actually [implements categories on the frontend](https://github.com/mattupstate/flask-security/blob/develop/flask_security/templates/security/_messages.html#L5)